pkg/spdk/engine.go

```diff
@@ -173,6 +174,12 @@ func (e *Engine) handleFrontend(spdkClient *spdkclient.Client, portCount int32,
 	}
 
 	nqn := helpertypes.GetNQN(e.Name)
+
+	e.log.Infof("Blindly stopping expose bdev for engine")
+	if err := spdkClient.StopExposeBdev(nqn); err != nil {
```

Comment:
ref: if not found, there will be no error returned.

[/pkg/spdk/client/advanced.go#L92-L94](https://github.com/longhorn/go-spdk-helper/blob/3b72caa5a1eac0e26f92db346009f5a9c5f7fa0e/pkg/spdk/client/advanced.go#L92-L94)
```go
	if subsystem == nil {
		return nil
	}
```